Classes/Integrations/LeagueOfLegendsIntegration.cs

```diff
@@ -53,7 +53,7 @@ internal class LeagueOfLegendsIntegration : Integration {
                     JsonElement allPlayers = root.GetProperty("allPlayers");
                     JsonElement currentPlayer = allPlayers
                         .EnumerateArray()
-                        .FirstOrDefault(playerElement => playerElement.GetProperty("summonerName").GetString() == username.Split('#')[0]);
```

Comment:
I don't play League much anymore so I don't know exactly what has changed, but shouldn't we be keeping this to handle Riot ID names? Or does that not matter?

Reply:
Yeah, I changed it to riotId. They updated it so that the summonerName now includes the # number.
But i think riotId is better to use.